server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -336,7 +340,9 @@ private void compactLocalityGroup(String lgName, Set<ByteSequence> columnFamilie
       boolean inclusive, FileSKVWriter mfw, CompactionStats majCStats)
       throws IOException, CompactionCanceledException {
     ArrayList<FileSKVIterator> readers = new ArrayList<>(filesToCompact.size());
-    try (TraceScope span = Trace.startSpan("compact")) {
+    Tracer tracer = TraceUtil.getTracer();
+    Span compactSpan = tracer.spanBuilder("FileCompactor::compact").startSpan();
```

Comment:
Our naming conventions are terribly inconsistent. I like them sometime like what you did here, but the rest of our code seems to follow very random naming. If we did try to reduce this boiler-plate, it might be easier to enforce a better naming convention whatever TraceUtil method is used to encapsulate that boiler-plate.

Reply:
I attempted to address this with the new TraceUtil.createSpan methods I added in 89a814b